packages/js/src/ui/components/elements/Preferences/Preferences.tsx

```diff
@@ -164,27 +159,9 @@ const PreferencesRow = (props: {
                 'workflowLabel',
                 'nt-text-base nt-font-semibold nt-text-start nt-flex nt-items-center nt-gap-1'
               )}
-              data-disabled={props.isCritical}
               data-localization={props.localizationKey}
               data-open={isOpen()}
             >
-              <Show when={props.isCritical}>
-                <Tooltip.Root>
-                  <Tooltip.Trigger
-                    asChild={(childProps) => (
-                      <span
-                        class={style('workflowLabelDisabled__icon', 'nt-text-foreground-alpha-600')}
-                        {...childProps}
-                      >
-                        <Lock />
-                      </span>
-                    )}
-                  />
-                  <Tooltip.Content data-localization="preferences.workflow.disabled.tooltip">
-                    {t('preferences.workflow.disabled.tooltip')}
-                  </Tooltip.Content>
-                </Tooltip.Root>
-              </Show>
```

Comment:
It no more fetches the critical workflows show we don't need this UI part to indicate a workflow is critical